### ISSUE TITLE
Song menu select freezes for 1-2 secs when switching map on HDD

### ISSUE THREAD

**Issue by 0loli** [ppy/osu#17460]
### Type

Performance

### Bug description

When I switch the map to another, the game freezes for 1-2 seconds. This behavior happens on all of my HDDs. If I use ssd it goes okay. 

I have not tested this on osu stable.

### Version

2022.319.0 

### Logs

database:
```
2022-03-26 10:35:52 [verbose]: ----------------------------------------------------------
2022-03-26 10:35:52 [verbose]: database Log for hidden (LogLevel: Verbose)
2022-03-26 10:35:52 [verbose]: Running osu 2022.319.0.0 on .NET 6.0.2
2022-03-26 10:35:52 [verbose]: Environment: Linux (Unix 5.15.26.1), 6 cores
2022-03-26 10:35:52 [verbose]: ----------------------------------------------------------
2022-03-26 10:35:52 [verbose]: [2a9ed] Beginning import...
2022-03-26 10:35:52 [verbose]: [2a9ed] [BeatmapOnlineLookupQueue] Cached local retrieval for IAMX - My Secret Friend feat. Imogen Heap (Raja) [Hard].
2022-03-26 10:35:52 [verbose]: [2a9ed] [BeatmapOnlineLookupQueue] Cached local retrieval for IAMX - My Secret Friend feat. Imogen Heap (Raja) [Another].
2022-03-26 10:35:52 [verbose]: [2a9ed] [BeatmapOnlineLookupQueue] Cached local retrieval for IAMX - My Secret Friend feat. Imogen Heap (Raja) [PandaHero's Insane].
2022-03-26 10:35:52 [verbose]: [2a9ed] Import successfully completed!
```

network
```
2022-03-26 10:59:47 [verbose]: ----------------------------------------------------------
2022-03-26 10:59:47 [verbose]: network Log for hidden (LogLevel: Verbose)
2022-03-26 10:59:47 [verbose]: Running osu 2022.319.0.0 on .NET 6.0.2
2022-03-26 10:59:47 [verbose]: Environment: Linux (Unix 5.15.26.1), 6 cores
2022-03-26 10:59:47 [verbose]: ----------------------------------------------------------
2022-03-26 10:59:47 [verbose]: Performing request osu.Game.Online.API.Requests.GetUserRequest
2022-03-26 10:59:48 [verbose]: Request to https://osu.ppy.sh/api/v2/me/ successfully completed!
2022-03-26 10:59:48 [verbose]: osu.Game.Online.API.Requests.GetUserRequest finished with response size of 7,463 bytes
2022-03-26 10:59:48 [verbose]: Performing request osu.Game.Online.API.Requests.GetFriendsRequest
2022-03-26 10:59:48 [verbose]: Request to https://osu.ppy.sh/api/v2/friends successfully completed!
2022-03-26 10:59:48 [verbose]: osu.Game.Online.API.Requests.GetFriendsRequest finished with response size of 2,186 bytes
2022-03-26 10:59:48 [verbose]: OnlineMultiplayerClient connecting...
2022-03-26 10:59:48 [verbose]: SpectatorClient connecting...
2022-03-26 10:59:48 [verbose]: Performing request osu.Game.Online.API.Requests.GetUserRequest
2022-03-26 10:59:49 [verbose]: Request to https://osu.ppy.sh/api/v2/users/hidden/osu?key=id successfully completed!
2022-03-26 10:59:49 [verbose]: osu.Game.Online.API.Requests.GetUserRequest finished with response size of 7,463 bytes
2022-03-26 10:59:49 [verbose]: Performing request osu.Game.Online.API.Requests.GetUserRequest
2022-03-26 10:59:49 [verbose]: Request to https://osu.ppy.sh/api/v2/users/hidden/taiko?key=id successfully completed!
2022-03-26 10:59:49 [verbose]: osu.Game.Online.API.Requests.GetUserRequest finished with response size of 7,270 bytes
2022-03-26 10:59:49 [verbose]: Performing request osu.Game.Online.API.Requests.GetUserRequest
2022-03-26 10:59:49 [verbose]: Request to https://api.github.com/repos/ppy/osu/releases/latest successfully completed!
2022-03-26 10:59:50 [verbose]: Request to https://osu.ppy.sh/api/v2/users/hidden/fruits?key=id successfully completed!
2022-03-26 10:59:50 [verbose]: osu.Game.Online.API.Requests.GetUserRequest finished with response size of 7,459 bytes
2022-03-26 10:59:50 [verbose]: Performing request osu.Game.Online.API.Requests.GetUserRequest
2022-03-26 10:59:50 [verbose]: Request to https://osu.ppy.sh/api/v2/users/hidden/mania?key=id successfully completed!
2022-03-26 10:59:50 [verbose]: osu.Game.Online.API.Requests.GetUserRequest finished with response size of 7,633 bytes
2022-03-26 10:59:50 [verbose]: Performing request osu.Game.Online.API.Requests.GetUpdatesRequest
2022-03-26 10:59:50 [verbose]: OnlineMultiplayerClient connected!
2022-03-26 10:59:51 [verbose]: Request to https://osu.ppy.sh/api/v2/chat/updates successfully completed!
2022-03-26 10:59:51 [verbose]: osu.Game.Online.API.Requests.GetUpdatesRequest finished with response size of 25,529 bytes
2022-03-26 10:59:51 [verbose]: Performing request osu.Game.Online.API.Requests.GetMessagesRequest
2022-03-26 10:59:51 [verbose]: SpectatorClient connected!
2022-03-26 10:59:51 [verbose]: Request to https://osu.ppy.sh/api/v2/chat/channels/5/messages successfully completed!
2022-03-26 10:59:51 [verbose]: osu.Game.Online.API.Requests.GetMessagesRequest finished with response size of 24,865 bytes
2022-03-26 10:59:51 [verbose]: Performing request osu.Game.Online.API.Requests.ListChannelsRequest
2022-03-26 10:59:51 [verbose]: Request to https://osu.ppy.sh/api/v2/chat/channels successfully completed!
2022-03-26 10:59:51 [verbose]: osu.Game.Online.API.Requests.ListChannelsRequest finished with response size of 5,505 bytes
2022-03-26 10:59:55 [verbose]: Performing request osu.Game.Online.API.Requests.GetSeasonalBackgroundsRequest
2022-03-26 10:59:56 [verbose]: Performing request osu.Game.Online.API.Requests.GetSeasonalBackgroundsRequest
2022-03-26 10:59:56 [verbose]: Request to https://osu.ppy.sh/api/v2/seasonal-backgrounds successfully completed!
2022-03-26 10:59:56 [verbose]: osu.Game.Online.API.Requests.GetSeasonalBackgroundsRequest finished with response size of 8,765 bytes
2022-03-26 10:59:56 [verbose]: Request to https://osu.ppy.sh/api/v2/seasonal-backgrounds successfully completed!
2022-03-26 10:59:56 [verbose]: osu.Game.Online.API.Requests.GetSeasonalBackgroundsRequest finished with response size of 8,765 bytes
2022-03-26 11:00:01 [verbose]: Request to https://a.ppy.sh/hidden?1jpeg successfully completed!
2022-03-26 11:00:01 [verbose]: Performing request osu.Game.Online.API.Requests.GetBeatmapRequest
2022-03-26 11:00:01 [verbose]: Request to https://osu.ppy.sh/api/v2/beatmaps/lookup successfully completed!
2022-03-26 11:00:01 [verbose]: osu.Game.Online.API.Requests.GetBeatmapRequest finished with response size of 2,915 bytes
2022-03-26 11:00:04 [verbose]: Performing request osu.Game.Online.API.Requests.GetBeatmapRequest
2022-03-26 11:00:05 [verbose]: Request to https://osu.ppy.sh/api/v2/beatmaps/lookup successfully completed!
2022-03-26 11:00:05 [verbose]: osu.Game.Online.API.Requests.GetBeatmapRequest finished with response size of 3,188 bytes
2022-03-26 11:00:07 [verbose]: Performing request osu.Game.Online.API.Requests.GetBeatmapRequest
2022-03-26 11:00:07 [verbose]: Request to https://osu.ppy.sh/api/v2/beatmaps/lookup successfully completed!
2022-03-26 11:00:07 [verbose]: osu.Game.Online.API.Requests.GetBeatmapRequest finished with response size of 3,250 bytes
2022-03-26 11:00:10 [verbose]: Performing request osu.Game.Online.API.Requests.GetBeatmapRequest
2022-03-26 11:00:11 [verbose]: Request to https://osu.ppy.sh/api/v2/beatmaps/lookup successfully completed!
2022-03-26 11:00:11 [verbose]: osu.Game.Online.API.Requests.GetBeatmapRequest finished with response size of 4,160 bytes
2022-03-26 11:00:12 [verbose]: Performing request osu.Game.Online.API.Requests.GetBeatmapRequest
2022-03-26 11:00:12 [verbose]: Request to https://osu.ppy.sh/api/v2/beatmaps/lookup successfully completed!
2022-03-26 11:00:12 [verbose]: osu.Game.Online.API.Requests.GetBeatmapRequest finished with response size of 3,078 bytes
2022-03-26 11:00:13 [verbose]: Performing request osu.Game.Online.API.Requests.GetBeatmapRequest
2022-03-26 11:00:13 [verbose]: Request to https://osu.ppy.sh/api/v2/beatmaps/lookup successfully completed!
2022-03-26 11:00:13 [verbose]: osu.Game.Online.API.Requests.GetBeatmapRequest finished with response size of 4,263 bytes
```

perfomance:
```
2022-03-26 10:59:47 [verbose]: ----------------------------------------------------------
2022-03-26 10:59:47 [verbose]: performance Log for hidden (LogLevel: Verbose)
2022-03-26 10:59:47 [verbose]: Running osu 2022.319.0.0 on .NET 6.0.2
2022-03-26 10:59:47 [verbose]: Environment: Linux (Unix 5.15.26.1), 6 cores
2022-03-26 10:59:47 [verbose]: ----------------------------------------------------------
2022-03-26 10:59:47 [verbose]: TextureAtlas initialised (1024x1024)
2022-03-26 10:59:47 [verbose]: TextureAtlas initialised (1024x1024)
2022-03-26 10:59:48 [verbose]: TextureAtlas size exceeded 1 time(s); generating new texture (1024x1024)
2022-03-26 10:59:48 [verbose]: TextureAtlas size exceeded 2 time(s); generating new texture (1024x1024)
2022-03-26 10:59:50 [verbose]: TextureAtlas size exceeded 1 time(s); generating new texture (1024x1024)
2022-03-26 10:59:50 [verbose]: TextureAtlas size exceeded 3 time(s); generating new texture (1024x1024)
2022-03-26 10:59:50 [verbose]: Texture requested (1400x310) which exceeds TextureStore's atlas size (1024x1024) - bypassing atlasing. Consider using LargeTextureStore.
2022-03-26 10:59:50 [verbose]: Texture requested (1800x600) which exceeds TextureStore's atlas size (1024x1024) - bypassing atlasing. Consider using LargeTextureStore.
2022-03-26 10:59:50 [verbose]: Texture requested (1400x310) which exceeds TextureStore's atlas size (1024x1024) - bypassing atlasing. Consider using LargeTextureStore.
2022-03-26 10:59:51 [verbose]: Texture requested (1366x768) which exceeds TextureStore's atlas size (1024x1024) - bypassing atlasing. Consider using LargeTextureStore.
2022-03-26 10:59:56 [verbose]: Texture requested (1295x1295) which exceeds TextureStore's atlas size (1024x1024) - bypassing atlasing. Consider using LargeTextureStore.
2022-03-26 10:59:56 [verbose]: Texture requested (1295x1295) which exceeds TextureStore's atlas size (1024x1024) - bypassing atlasing. Consider using LargeTextureStore.
2022-03-26 10:59:56 [verbose]: TextureAtlas size exceeded 2 time(s); generating new texture (1024x1024)
2022-03-26 11:00:13 [verbose]: TextureAtlas size exceeded 3 time(s); generating new texture (1024x1024)
```

runtime
```
2022-03-26 10:59:44 [verbose]: ----------------------------------------------------------
2022-03-26 10:59:44 [verbose]: runtime Log for hidden (LogLevel: Verbose)
2022-03-26 10:59:44 [verbose]: Running osu 2022.319.0.0 on .NET 6.0.2
2022-03-26 10:59:44 [verbose]: Environment: Linux (Unix 5.15.26.1), 6 cores
2022-03-26 10:59:44 [verbose]: ----------------------------------------------------------
2022-03-26 10:59:44 [verbose]: Detect: Searching for tablets...
2022-03-26 10:59:44 [verbose]: Detect: No tablets were detected.
2022-03-26 10:59:45 [verbose]: Connected MIDI device: Midi Through Port-0
2022-03-26 10:59:45 [verbose]: GL Initialized
2022-03-26 10:59:45 [verbose]: GL Version:                 4.6.0 NVIDIA 510.54
2022-03-26 10:59:45 [verbose]: GL Renderer:                NVIDIA `hidden`
2022-03-26 10:59:45 [verbose]: GL Shader Language version: 4.60 NVIDIA
2022-03-26 10:59:45 [verbose]: GL Vendor:                  NVIDIA Corporation
2022-03-26 10:59:45 [verbose]: GL Extensions:              GL_AMD_multi_draw_indirect GL_AMD_seamless_cubemap_per_texture GL_AMD_vertex_shader_viewport_index GL_AMD_vertex_shader_layer GL_ARB_arrays_of_arrays GL_ARB_base_instance GL_ARB_bindless_texture GL_ARB_blend_func_extended GL_ARB_buffer_storage GL_ARB_clear_buffer_object GL_ARB_clear_texture GL_ARB_clip_control GL_ARB_color_buffer_float GL_ARB_compatibility GL_ARB_compressed_texture_pixel_storage GL_ARB_conservative_depth GL_ARB_compute_shader GL_ARB_compute_variable_group_size GL_ARB_conditional_render_inverted GL_ARB_copy_buffer GL_ARB_copy_image GL_ARB_cull_distance GL_ARB_debug_output GL_ARB_depth_buffer_float GL_ARB_depth_clamp GL_ARB_depth_texture GL_ARB_derivative_control GL_ARB_direct_state_access GL_ARB_draw_buffers GL_ARB_draw_buffers_blend GL_ARB_draw_indirect GL_ARB_draw_elements_base_vertex GL_ARB_draw_instanced GL_ARB_enhanced_layouts GL_ARB_ES2_compatibility GL_ARB_ES3_compatibility GL_ARB_ES3_1_compatibility GL_ARB_ES3_2_compatibility GL_ARB_explicit_attrib_location GL_ARB_explicit_uniform_location GL_ARB_fragment_coord_conventions GL_ARB_fragment_layer_viewport GL_ARB_fragment_program GL_ARB_fragment_program_shadow GL_ARB_fragment_shader GL_ARB_fragment_shader_interlock GL_ARB_framebuffer_no_attachments GL_ARB_framebuffer_object GL_ARB_framebuffer_sRGB GL_ARB_geometry_shader4 GL_ARB_get_program_binary GL_ARB_get_texture_sub_image GL_ARB_gl_spirv GL_ARB_gpu_shader5 GL_ARB_gpu_shader_fp64 GL_ARB_gpu_shader_int64 GL_ARB_half_float_pixel GL_ARB_half_float_vertex GL_ARB_imaging GL_ARB_indirect_parameters GL_ARB_instanced_arrays GL_ARB_internalformat_query GL_ARB_internalformat_query2 GL_ARB_invalidate_subdata GL_ARB_map_buffer_alignment GL_ARB_map_buffer_range GL_ARB_multi_bind GL_ARB_multi_draw_indirect GL_ARB_multisample GL_ARB_multitexture GL_ARB_occlusion_query GL_ARB_occlusion_query2 GL_ARB_parallel_shader_compile GL_ARB_pipeline_statistics_query GL_ARB_pixel_buffer_object GL_ARB_point_parameters GL_ARB_point_sprite GL_ARB_polygon_offset_clamp GL_ARB_post_depth_coverage GL_ARB_program_interface_query GL_ARB_provoking_vertex GL_ARB_query_buffer_object GL_ARB_robust_buffer_access_behavior GL_ARB_robustness GL_ARB_sample_locations GL_ARB_sample_shading GL_ARB_sampler_objects GL_ARB_seamless_cube_map GL_ARB_seamless_cubemap_per_texture GL_ARB_separate_shader_objects GL_ARB_shader_atomic_counter_ops GL_ARB_shader_atomic_counters GL_ARB_shader_ballot GL_ARB_shader_bit_encoding GL_ARB_shader_clock GL_ARB_shader_draw_parameters GL_ARB_shader_group_vote GL_ARB_shader_image_load_store GL_ARB_shader_image_size GL_ARB_shader_objects GL_ARB_shader_precision GL_ARB_shader_storage_buffer_object GL_ARB_shader_subroutine GL_ARB_shader_texture_image_samples GL_ARB_shader_texture_lod GL_ARB_shading_language_100 GL_ARB_shader_viewport_layer_array GL_ARB_shading_language_420pack GL_ARB_shading_language_include GL_ARB_shading_language_packing GL_ARB_shadow GL_ARB_sparse_buffer GL_ARB_sparse_texture GL_ARB_sparse_texture2 GL_ARB_sparse_texture_clamp GL_ARB_spirv_extensions GL_ARB_stencil_texturing GL_ARB_sync GL_ARB_tessellation_shader GL_ARB_texture_barrier GL_ARB_texture_border_clamp GL_ARB_texture_buffer_object GL_ARB_texture_buffer_object_rgb32 GL_ARB_texture_buffer_range GL_ARB_texture_compression GL_ARB_texture_compression_bptc GL_ARB_texture_compression_rgtc GL_ARB_texture_cube_map GL_ARB_texture_cube_map_array GL_ARB_texture_env_add GL_ARB_texture_env_combine GL_ARB_texture_env_crossbar GL_ARB_texture_env_dot3 GL_ARB_texture_filter_anisotropic GL_ARB_texture_filter_minmax GL_ARB_texture_float GL_ARB_texture_gather GL_ARB_texture_mirror_clamp_to_edge GL_ARB_texture_mirrored_repeat GL_ARB_texture_multisample GL_ARB_texture_non_power_of_two GL_ARB_texture_query_levels GL_ARB_texture_query_lod GL_ARB_texture_rectangle GL_ARB_texture_rg GL_ARB_texture_rgb10_a2ui GL_ARB_texture_stencil8 GL_ARB_texture_storage GL_ARB_texture_storage_multisample GL_ARB_texture_swizzle GL_ARB_texture_view GL_ARB_timer_query GL_ARB_transform_feedback2 GL_ARB_transform_feedback3 GL_ARB_transform_feedback_instanced GL_ARB_transform_feedback_overflow_query GL_ARB_transpose_matrix GL_ARB_uniform_buffer_object GL_ARB_vertex_array_bgra GL_ARB_vertex_array_object GL_ARB_vertex_attrib_64bit GL_ARB_vertex_attrib_binding GL_ARB_vertex_buffer_object GL_ARB_vertex_program GL_ARB_vertex_shader GL_ARB_vertex_type_10f_11f_11f_rev GL_ARB_vertex_type_2_10_10_10_rev GL_ARB_viewport_array GL_ARB_window_pos GL_ATI_draw_buffers GL_ATI_texture_float GL_ATI_texture_mirror_once GL_S3_s3tc GL_EXT_texture_env_add GL_EXT_abgr GL_EXT_bgra GL_EXT_bindable_uniform GL_EXT_blend_color GL_EXT_blend_equation_separate GL_EXT_blend_func_separate GL_EXT_blend_minmax GL_EXT_blend_subtract GL_EXT_compiled_vertex_array GL_EXT_Cg_shader GL_EXT_depth_bounds_test GL_EXT_direct_state_access GL_EXT_draw_buffers2 GL_EXT_draw_instanced GL_EXT_draw_range_elements GL_EXT_fog_coord GL_EXT_framebuffer_blit GL_EXT_framebuffer_multisample GL_EXTX_framebuffer_mixed_formats GL_EXT_framebuffer_multisample_blit_scaled GL_EXT_framebuffer_object GL_EXT_framebuffer_sRGB GL_EXT_geometry_shader4 GL_EXT_gpu_program_parameters GL_EXT_gpu_shader4 GL_EXT_multi_draw_arrays GL_EXT_multiview_texture_multisample GL_EXT_multiview_timer_query GL_EXT_packed_depth_stencil GL_EXT_packed_float GL_EXT_packed_pixels GL_EXT_pixel_buffer_object GL_EXT_point_parameters GL_EXT_polygon_offset_clamp GL_EXT_post_depth_coverage GL_EXT_provoking_vertex GL_EXT_raster_multisample GL_EXT_rescale_normal GL_EXT_secondary_color GL_EXT_separate_shader_objects GL_EXT_separate_specular_color GL_EXT_shader_image_load_formatted GL_EXT_shader_image_load_store GL_EXT_shader_integer_mix GL_EXT_shadow_funcs GL_EXT_sparse_texture2 GL_EXT_stencil_two_side GL_EXT_stencil_wrap GL_EXT_texture3D GL_EXT_texture_array GL_EXT_texture_buffer_object GL_EXT_texture_compression_dxt1 GL_EXT_texture_compression_latc GL_EXT_texture_compression_rgtc GL_EXT_texture_compression_s3tc GL_EXT_texture_cube_map GL_EXT_texture_edge_clamp GL_EXT_texture_env_combine GL_EXT_texture_env_dot3 GL_EXT_texture_filter_anisotropic GL_EXT_texture_filter_minmax GL_EXT_texture_integer GL_EXT_texture_lod GL_EXT_texture_lod_bias GL_EXT_texture_mirror_clamp GL_EXT_texture_object GL_EXT_texture_shadow_lod GL_EXT_texture_shared_exponent GL_EXT_texture_sRGB GL_EXT_texture_sRGB_R8 GL_EXT_texture_sRGB_decode GL_EXT_texture_storage GL_EXT_texture_swizzle GL_EXT_timer_query GL_EXT_transform_feedback2 GL_EXT_vertex_array GL_EXT_vertex_array_bgra GL_EXT_vertex_attrib_64bit GL_EXT_window_rectangles GL_EXT_x11_sync_object GL_EXT_import_sync_object GL_NV_robustness_video_memory_purge GL_IBM_rasterpos_clip GL_IBM_texture_mirrored_repeat GL_KHR_context_flush_control GL_KHR_debug GL_EXT_memory_object GL_EXT_memory_object_fd GL_NV_memory_object_sparse GL_KHR_parallel_shader_compile GL_KHR_no_error GL_KHR_robust_buffer_access_behavior GL_KHR_robustness GL_EXT_semaphore GL_EXT_semaphore_fd GL_NV_timeline_semaphore GL_KHR_shader_subgroup GL_KTX_buffer_region GL_NV_alpha_to_coverage_dither_control GL_NV_bindless_multi_draw_indirect GL_NV_bindless_multi_draw_indirect_count GL_NV_bindless_texture GL_NV_blend_equation_advanced GL_NV_blend_equation_advanced_coherent GL_NVX_blend_equation_advanced_multi_draw_buffers GL_NV_blend_minmax_factor GL_NV_blend_square GL_NV_clip_space_w_scaling GL_NV_command_list GL_NV_compute_program5 GL_NV_compute_shader_derivatives GL_NV_conditional_render GL_NV_conservative_raster GL_NV_conservative_raster_dilate GL_NV_conservative_raster_pre_snap GL_NV_conservative_raster_pre_snap_triangles GL_NV_conservative_raster_underestimation GL_NV_copy_depth_to_color GL_NV_copy_image GL_NV_depth_buffer_float GL_NV_depth_clamp GL_NV_draw_texture GL_NV_draw_vulkan_image GL_NV_ES1_1_compatibility GL_NV_ES3_1_compatibility GL_NV_explicit_multisample GL_NV_feature_query GL_NV_fence GL_NV_fill_rectangle GL_NV_float_buffer GL_NV_fog_distance GL_NV_fragment_coverage_to_color GL_NV_fragment_program GL_NV_fragment_program_option GL_NV_fragment_program2 GL_NV_fragment_shader_barycentric GL_NV_fragment_shader_interlock GL_NV_framebuffer_mixed_samples GL_NV_framebuffer_multisample_coverage GL_NV_geometry_shader4 GL_NV_geometry_shader_passthrough GL_NV_gpu_program4 GL_NV_internalformat_sample_query GL_NV_gpu_program4_1 GL_NV_gpu_program5 GL_NV_gpu_program5_mem_extended GL_NV_gpu_program_fp64 GL_NV_gpu_shader5 GL_NV_half_float GL_NV_light_max_exponent GL_NV_memory_attachment GL_NV_mesh_shader GL_NV_multisample_coverage GL_NV_multisample_filter_hint GL_NV_occlusion_query GL_NV_packed_depth_stencil GL_NV_parameter_buffer_object GL_NV_parameter_buffer_object2 GL_NV_path_rendering GL_NV_path_rendering_shared_edge GL_NV_pixel_data_range GL_NV_point_sprite GL_NV_primitive_restart GL_NV_query_resource GL_NV_query_resource_tag GL_NV_register_combiners GL_NV_register_combiners2 GL_NV_representative_fragment_test GL_NV_sample_locations GL_NV_sample_mask_override_coverage GL_NV_scissor_exclusive GL_NV_shader_atomic_counters GL_NV_shader_atomic_float GL_NV_shader_atomic_float64 GL_NV_shader_atomic_fp16_vector GL_NV_shader_atomic_int64 GL_NV_shader_buffer_load GL_NV_shader_storage_buffer_object GL_NV_shader_subgroup_partitioned GL_NV_shader_texture_footprint GL_NV_shading_rate_image GL_NV_stereo_view_rendering GL_NV_texgen_reflection GL_NV_texture_barrier GL_NV_texture_compression_vtc GL_NV_texture_env_combine4 GL_NV_texture_multisample GL_NV_texture_rectangle GL_NV_texture_rectangle_compressed GL_NV_texture_shader GL_NV_texture_shader2 GL_NV_texture_shader3 GL_NV_transform_feedback GL_NV_transform_feedback2 GL_NV_uniform_buffer_unified_memory GL_NV_vdpau_interop GL_NV_vdpau_interop2 GL_NV_vertex_array_range GL_NV_vertex_array_range2 GL_NV_vertex_attrib_integer_64bit GL_NV_vertex_buffer_unified_memory GL_NV_vertex_program GL_NV_vertex_program1_1 GL_NV_vertex_program2 GL_NV_vertex_program2_option GL_NV_vertex_program3 GL_NV_viewport_array2 GL_NV_viewport_swizzle GL_NVX_conditional_render GL_NV_gpu_multicast GL_NVX_progress_fence GL_NVX_gpu_memory_info GL_NVX_nvenc_interop GL_NV_shader_thread_group GL_NV_shader_thread_shuffle GL_KHR_blend_equation_advanced GL_KHR_blend_equation_advanced_coherent GL_OVR_multiview GL_OVR_multiview2 GL_SGIS_generate_mipmap GL_SGIS_texture_lod GL_SGIX_depth_texture GL_SGIX_shadow GL_SUN_slice_accum
2022-03-26 10:59:45 [verbose]: Host execution state changed to Running
2022-03-26 10:59:45 [verbose]: Execution mode changed to MultiThreaded
2022-03-26 10:59:45 [verbose]: 🔈 BASS initialised
2022-03-26 10:59:45 [verbose]: BASS version:           2.4.16.7
2022-03-26 10:59:45 [verbose]: BASS FX version:        2.4.12.6
2022-03-26 10:59:45 [verbose]: BASS MIX version:       2.4.11.1
2022-03-26 10:59:45 [verbose]: Device:                 No sound
2022-03-26 10:59:45 [verbose]: Driver:
2022-03-26 10:59:45 [verbose]: Update period:          5 ms
2022-03-26 10:59:45 [verbose]: Device buffer length:   10 ms
2022-03-26 10:59:45 [verbose]: Playback buffer length: 100 ms
2022-03-26 10:59:46 [verbose]: 🔈 BASS initialised
2022-03-26 10:59:46 [verbose]: BASS version:           2.4.16.7
2022-03-26 10:59:46 [verbose]: BASS FX version:        2.4.12.6
2022-03-26 10:59:46 [verbose]: BASS MIX version:       2.4.11.1
2022-03-26 10:59:46 [verbose]: Device:                 Default
2022-03-26 10:59:46 [verbose]: Driver:                 default
2022-03-26 10:59:46 [verbose]: Update period:          5 ms
2022-03-26 10:59:46 [verbose]: Device buffer length:   10 ms
2022-03-26 10:59:46 [verbose]: Playback buffer length: 100 ms
2022-03-26 10:59:46 [verbose]: Beginning realm file store cleanup
2022-03-26 10:59:46 [verbose]: Finished realm file store cleanup (0 of 4395 deleted)
2022-03-26 10:59:47 [verbose]: Game-wide working beatmap updated to please load a beatmap! - no beatmaps available!
2022-03-26 10:59:48 [verbose]: Loading VersionManager...
2022-03-26 10:59:48 [verbose]: Opened realm "/home/hidden/.local/share/osu/client.realm" at version 14
2022-03-26 10:59:48 [verbose]: Loaded VersionManager!
2022-03-26 10:59:48 [verbose]: Loading OsuLogo...
2022-03-26 10:59:48 [verbose]: Loaded OsuLogo!
2022-03-26 10:59:48 [verbose]: Loading Toolbar...
2022-03-26 10:59:48 [verbose]: 📺 OsuScreenStack#412(depth:1) loading Loader#549
2022-03-26 10:59:48 [verbose]: 📺 OsuScreenStack#412(depth:1) entered Loader#549
2022-03-26 10:59:49 [verbose]: Loaded Toolbar!
2022-03-26 10:59:49 [verbose]: Loading VolumeOverlay...
2022-03-26 10:59:49 [verbose]: Loaded VolumeOverlay!
2022-03-26 10:59:49 [verbose]: Loading OnScreenDisplay...
2022-03-26 10:59:49 [verbose]: Loaded OnScreenDisplay!
2022-03-26 10:59:49 [verbose]: Loading NotificationOverlay...
2022-03-26 10:59:49 [verbose]: Loaded NotificationOverlay!
2022-03-26 10:59:49 [verbose]: Loading CollectionManager...
2022-03-26 10:59:49 [verbose]: Loaded CollectionManager!
2022-03-26 10:59:49 [verbose]: Loading LegacyImportManager...
2022-03-26 10:59:49 [verbose]: Loaded LegacyImportManager!
2022-03-26 10:59:49 [verbose]: Loading ScreenshotManager...
2022-03-26 10:59:49 [verbose]: Loaded ScreenshotManager!
2022-03-26 10:59:49 [verbose]: Loading SimpleUpdateManager...
2022-03-26 10:59:49 [verbose]: Loaded SimpleUpdateManager!
2022-03-26 10:59:49 [verbose]: Loading ManageCollectionsDialog...
2022-03-26 10:59:49 [verbose]: Loaded ManageCollectionsDialog!
2022-03-26 10:59:49 [verbose]: Loading BeatmapListingOverlay...
2022-03-26 10:59:49 [verbose]: 📺 OsuScreenStack#412(depth:1) suspended Loader#549 (waiting on Disclaimer#488)
2022-03-26 10:59:49 [verbose]: 📺 OsuScreenStack#412(depth:2) entered Disclaimer#488
2022-03-26 10:59:50 [verbose]: Loaded BeatmapListingOverlay!
2022-03-26 10:59:50 [verbose]: Loading DashboardOverlay...
2022-03-26 10:59:50 [verbose]: Loaded DashboardOverlay!
2022-03-26 10:59:50 [verbose]: Loading NewsOverlay...
2022-03-26 10:59:50 [verbose]: Loaded NewsOverlay!
2022-03-26 10:59:50 [verbose]: Loading RankingsOverlay...
2022-03-26 10:59:50 [verbose]: Loaded RankingsOverlay!
2022-03-26 10:59:50 [verbose]: Loading ChannelManager...
2022-03-26 10:59:50 [verbose]: Chat is now polling every 60000 ms
2022-03-26 10:59:50 [verbose]: Loaded ChannelManager!
2022-03-26 10:59:50 [verbose]: Loading ChatOverlay...
2022-03-26 10:59:50 [verbose]: Loaded ChatOverlay!
2022-03-26 10:59:50 [verbose]: Loading MessageNotifier...
2022-03-26 10:59:50 [verbose]: Loaded MessageNotifier!
2022-03-26 10:59:50 [verbose]: Loading SettingsOverlay...
2022-03-26 10:59:50 [verbose]: Loaded SettingsOverlay!
2022-03-26 10:59:50 [verbose]: Loading ChangelogOverlay...
2022-03-26 10:59:50 [verbose]: Loaded ChangelogOverlay!
2022-03-26 10:59:50 [verbose]: Loading UserProfileOverlay...
2022-03-26 10:59:50 [verbose]: Loaded UserProfileOverlay!
2022-03-26 10:59:50 [verbose]: Loading BeatmapSetOverlay...
2022-03-26 10:59:51 [verbose]: Loaded BeatmapSetOverlay!
2022-03-26 10:59:51 [verbose]: Loading WikiOverlay...
2022-03-26 10:59:51 [verbose]: Loaded WikiOverlay!
2022-03-26 10:59:51 [verbose]: Loading SkinEditorOverlay...
2022-03-26 10:59:51 [verbose]: Loaded SkinEditorOverlay!
2022-03-26 10:59:51 [verbose]: Loading LoginOverlay...
2022-03-26 10:59:51 [verbose]: Loaded LoginOverlay!
2022-03-26 10:59:51 [verbose]: Loading NowPlayingOverlay...
2022-03-26 10:59:51 [verbose]: Loaded NowPlayingOverlay!
2022-03-26 10:59:51 [verbose]: Loading AccountCreationOverlay...
2022-03-26 10:59:51 [verbose]: 📺 ScreenStack#313(depth:1) scheduling push ScreenWelcome#592
2022-03-26 10:59:51 [verbose]: Loaded AccountCreationOverlay!
2022-03-26 10:59:51 [verbose]: Loading DialogOverlay...
2022-03-26 10:59:51 [verbose]: Loaded DialogOverlay!
2022-03-26 10:59:51 [verbose]: Loading HighPerformanceSession...
2022-03-26 10:59:51 [verbose]: Loaded HighPerformanceSession!
2022-03-26 10:59:55 [verbose]: 📺 OsuScreenStack#412(depth:2) suspended Disclaimer#488 (waiting on IntroTriangles#610)
2022-03-26 10:59:55 [verbose]: 📺 OsuScreenStack#412(depth:3) entered IntroTriangles#610
2022-03-26 10:59:55 [verbose]: 📺 BackgroundScreenStack#400(depth:1) loading BackgroundScreenDefault#440
2022-03-26 10:59:55 [verbose]: Game-wide working beatmap updated to cYsmix - triangles (peppy) [peppy]
2022-03-26 10:59:56 [verbose]: 📺 BackgroundScreenStack#400(depth:1) entered BackgroundScreenDefault#440
2022-03-26 10:59:56 [verbose]: decoupled ruleset transferred ("" -> "osu!")
2022-03-26 10:59:59 [verbose]: 📺 OsuScreenStack#412(depth:3) suspended IntroTriangles#610 (waiting on MainMenu#155)
2022-03-26 10:59:59 [verbose]: 📺 OsuScreenStack#412(depth:4) entered MainMenu#155
2022-03-26 11:00:00 [verbose]: ButtonSystem's state changed from Initial to TopLevel
2022-03-26 11:00:00 [verbose]: ButtonSystem's state changed from TopLevel to Play
2022-03-26 11:00:00 [verbose]: 📺 OsuScreenStack#412(depth:4) suspended MainMenu#155 (waiting on PlaySongSelect#585)
2022-03-26 11:00:00 [verbose]: ButtonSystem's state changed from Play to EnteringMode
2022-03-26 11:00:00 [verbose]: 📺 OsuScreenStack#412(depth:5) entered PlaySongSelect#585
2022-03-26 11:00:00 [verbose]: 📺 BackgroundScreenStack#400(depth:2) loading BackgroundScreenBeatmap#447
2022-03-26 11:00:01 [verbose]: 📺 BackgroundScreenStack#400(depth:2) suspended BackgroundScreenDefault#440 (waiting on BackgroundScreenBeatmap#447)
2022-03-26 11:00:01 [verbose]: 📺 BackgroundScreenStack#400(depth:2) entered BackgroundScreenBeatmap#447
2022-03-26 11:00:01 [verbose]: updating selection with beatmap:cb4dd751-035a-4be4-8289-65bb94c32546 ruleset:osu
2022-03-26 11:00:01 [verbose]: beatmap changed from "cYsmix - triangles (peppy) [peppy]" to "MAN WITH A MISSION - database feat.TAKUMA(10-FEET) (Kasuga) [Anre's Easy]"
2022-03-26 11:00:01 [verbose]: Game-wide working beatmap updated to MAN WITH A MISSION - database feat.TAKUMA(10-FEET) (Kasuga) [Anre's Easy]
2022-03-26 11:00:01 [verbose]: Song select working beatmap updated to MAN WITH A MISSION - database feat.TAKUMA(10-FEET) (Kasuga) [Anre's Easy]
2022-03-26 11:00:03 [verbose]: updating selection with beatmap:7905105f-12a1-41dd-b019-6628cb6b785e ruleset:osu
2022-03-26 11:00:03 [verbose]: beatmap changed from "MAN WITH A MISSION - database feat.TAKUMA(10-FEET) (Kasuga) [Anre's Easy]" to "Stan SB - Dead (ProfessionalBox) [Rebirth]"
2022-03-26 11:00:03 [verbose]: Game-wide working beatmap updated to Stan SB - Dead (ProfessionalBox) [Rebirth]
2022-03-26 11:00:03 [verbose]: Song select working beatmap updated to Stan SB - Dead (ProfessionalBox) [Rebirth]
2022-03-26 11:00:05 [verbose]: updating selection with beatmap:b39f9996-6724-4abd-be0a-d57b27f8cef8 ruleset:osu
2022-03-26 11:00:05 [verbose]: beatmap changed from "Stan SB - Dead (ProfessionalBox) [Rebirth]" to "Linkin Park - Living Things (384059043) [Marathon Lite]"
2022-03-26 11:00:05 [verbose]: Game-wide working beatmap updated to Linkin Park - Living Things (384059043) [Marathon Lite]
2022-03-26 11:00:05 [verbose]: Song select working beatmap updated to Linkin Park - Living Things (384059043) [Marathon Lite]
2022-03-26 11:00:10 [verbose]: updating selection with beatmap:63133449-67b5-4764-8fc8-42ce053e580c ruleset:osu
2022-03-26 11:00:10 [verbose]: beatmap changed from "Linkin Park - Living Things (384059043) [Marathon Lite]" to "Dzeko & Torres - 2013 In 10 Minutes (Aleks719) [Marathon]"
2022-03-26 11:00:10 [verbose]: Game-wide working beatmap updated to Dzeko & Torres - 2013 In 10 Minutes (Aleks719) [Marathon]
2022-03-26 11:00:10 [verbose]: Song select working beatmap updated to Dzeko & Torres - 2013 In 10 Minutes (Aleks719) [Marathon]
2022-03-26 11:00:11 [verbose]: updating selection with beatmap:a9930870-03b9-40c5-a90a-99328162fcd0 ruleset:osu
2022-03-26 11:00:11 [verbose]: beatmap changed from "Dzeko & Torres - 2013 In 10 Minutes (Aleks719) [Marathon]" to "Nhato - Miss You (HanzeR) [Selentia]"
2022-03-26 11:00:11 [verbose]: Game-wide working beatmap updated to Nhato - Miss You (HanzeR) [Selentia]
2022-03-26 11:00:11 [verbose]: Song select working beatmap updated to Nhato - Miss You (HanzeR) [Selentia]
2022-03-26 11:00:12 [verbose]: updating selection with beatmap:33bc8966-c31e-4f11-93e6-05a73c2c4b02 ruleset:osu
2022-03-26 11:00:12 [verbose]: beatmap changed from "Nhato - Miss You (HanzeR) [Selentia]" to "nao - Songs Compilation ([Minakami Yuki]) [naorathon]"
2022-03-26 11:00:12 [verbose]: Game-wide working beatmap updated to nao - Songs Compilation ([Minakami Yuki]) [naorathon]
2022-03-26 11:00:12 [verbose]: Song select working beatmap updated to nao - Songs Compilation ([Minakami Yuki]) [naorathon]
2022-03-26 11:00:20 [verbose]: 📺 BackgroundScreenStack#400(depth:1) exit from BackgroundScreenBeatmap#447
2022-03-26 11:00:20 [verbose]: 📺 BackgroundScreenStack#400(depth:1) resume to BackgroundScreenDefault#440
2022-03-26 11:00:20 [verbose]: 📺 OsuScreenStack#412(depth:4) exit from PlaySongSelect#585
2022-03-26 11:00:20 [verbose]: 📺 OsuScreenStack#412(depth:4) resume to MainMenu#155
2022-03-26 11:00:20 [verbose]: decoupled ruleset transferred ("" -> "osu!")
2022-03-26 11:00:20 [verbose]: ButtonSystem's state changed from EnteringMode to TopLevel
2022-03-26 11:00:21 [verbose]: ButtonSystem's state changed from TopLevel to Exit
2022-03-26 11:00:21 [verbose]: 📺 OsuScreenStack#412(depth:3) exit from MainMenu#155
2022-03-26 11:00:21 [verbose]: 📺 OsuScreenStack#412(depth:3) resume to IntroTriangles#610
2022-03-26 11:00:21 [verbose]: Host execution state changed to Stopping
2022-03-26 11:00:21 [verbose]: Host execution state changed to Stopped
```

sensitive information has been hidden.

**Comment by smoogipoo** [comment]
Last time I had a lead on this it was due to track loading (which I believe is still synchronous). May still be relevant.

**Comment by 0loli** [comment]
> Last time I had a lead on this it was due to track loading (which I believe is still synchronous). May still be relevant.
I think so too. It should be definitely fixed by loading the track asynchronously.

**Comment by Sublimelime** [comment]
Duplicate of #5649 ?

**Comment by bdach** [comment]
Closed as duplicate.
